components/builder-depot/doc/api.raml

```diff
@@ -196,9 +196,34 @@ securitySchemes:
                                         "origin_id": "77731431660388352"
                                     }
 /pkgs:
+    /search:
+        /{query}:
+            get:
+                description: Search for packages with a query string
+                queryParameters:
+                    distinct:
+                        description: Whether to show a distinct list of packages or not
+                        type: boolean
+                        required: false
+                        default: false
+                        example: true
+                responses:
+                    200:
+                        description: Packages were found and fit on one page
+                    204:
+                        description: Packages were found and require pagination
```

Comment:
Sorry I wasn't very clear in my comment.

https://github.com/habitat-sh/habitat/blob/master/components/builder-depot/src/server.rs#L1483

That line shows how this API endpoint supports pagination.  I've understood the purpose of the API docs in the `builder-depot` component to be documenting the capabilities of the server.  Since the server is capable of pagination on this endpoint, I included it in the docs.

It's entirely possible I've misunderstood the purpose of these API docs though 😁